src/helpers/graphConfig.builder.helper.ts

```diff
@@ -37,4 +39,19 @@ export const buildGraphConfig = (
     };
 
     return graphToSave;
+};
+
+export const validateGraph = (graph: Graph) => {
+    const graphType = graph.template?.type;
+    const mustHaveElements: any[] = [graph.title, graphType];
```

Comment:
do u have to specify that it's any[]? cant typescript infer it on it's own?

Reply:
yes becuase it thinks that this is array of string but then I added string[] so it is confusing for him